src/components/BoxForm/DropZone/DropZone.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/prop-types */
+import React, { useMemo, useState, useEffect } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { CloseIcon } from '@chakra-ui/icons';
+import './DropZone.css';
+
+function DropZone({ setFiles }) {
+  const { getRootProps, getInputProps, isDragAccept, isDragReject, acceptedFiles, open } =
+    useDropzone({
+      noClick: true,
+      noKeyboard: true,
+      accept: 'image/jpeg, image/jpg, image/png',
+      maxFiles: 1,
+    });
+  const [uploaded, setUploaded] = useState(false);
+
+  useEffect(() => {
+    if (acceptedFiles.length > 0) {
+      setUploaded(true);
+      setFiles(acceptedFiles);
+    }
+  });
+
+  const dropzoneBox = useMemo(() => {
```

Comment:
What is the purpose of the useMemo() hook here? It seems like we are simply conditionally rendering the classname based on whether drag/drop is accepted or rejected. From what I'm reading, useMemo() is used for expensive functions.
https://dmitripavlutin.com/react-usememo-hook/

Reply:
The useMemo is used to style the drop zone depending on the file's type. The box will display a red border if you drag an inappropriate file type and a green border if you drag the correct type